### PR TITLE
Feature/assignment 2

### PR DESCRIPTION
**Pull request by nikhil1-successive**

### CODE REVIEW

File: Assignment-2/Ques3.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 function getName(){
     return "Nikhil"
 }
-function printname(name=getName()){
+function printName(name=getName()){
     console.log("Hii" + " "+name)
 }
-printname()
\ No newline at end of file
+printName()
```
Comment (shahbaz-successive) [inline Assignment-2/Ques3.js:10]: Pass function while calling it not while initializing

File: Assignment-2/Ques5.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Write a function expression that takes in two numbers and returns their sum.
 
-function add(n1,n2){
+function returnSum(n1,n2){
```
Comment (shahbaz-successive) [inline Assignment-2/Ques5.js:3]: Make use of more better function name like getSum

File: Assignment-2/Ques7.js [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
+    return "Hii" + " "+name  
 }
-console.log(f())
+console.log(printName())
 
 // Write a function expression that takes in a number and returns its square. USING ARROW FUNCTION
 
-const sqaure=(n)=>n*n
-console.log(sqaure(5))
+const returnSqaure=(number)=>number*number
```
Comment (shahbaz-successive) [inline Assignment-2/Ques7.js:14]: Update function name to getSquare

File: Assignment-2/Ques7.js [hunk 4]
```diff
@@ -1,29 +1,30 @@
...
+console.log(returnSqaure(5))
 
 // Write a function expression that takes in two numbers and returns their sum. USING ARROW FUNCTION
 
-const sum=(a,b)=>a+b;
-console.log(sum(3,4))
+const returnSum=(number1,number2)=>number1+number2
```
Comment (shahbaz-successive) [inline Assignment-2/Ques7.js:19]: Update function name to getSum

File: Assignment-2/Ques4.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 
 // Write a function expression that takes in a number and returns its square.
 
-function square(n){
+function returnSquare(n){
```
Comment (shahbaz-successive) [inline Assignment-2/Ques4.js:4]: Update function name to getSquare

**Review by shahbaz-successive — CHANGES_REQUESTED** [review]
Feedback changes required

File: Assignment-2/Ques3.js (hunk 1 above)
Comment (nikhil1-successive) [reply Assignment-2/Ques3.js:10]: Done

File: Assignment-2/Ques7.js (hunk 3 above)
Comment (nikhil1-successive) [reply Assignment-2/Ques7.js:14]: Done

File: Assignment-2/Ques5.js (hunk 2 above)
Comment (nikhil1-successive) [reply Assignment-2/Ques5.js]: Done

File: Assignment-2/Ques4.js (hunk 5 above)
Comment (nikhil1-successive) [reply Assignment-2/Ques4.js]: Done

File: Assignment-2/Ques7.js [hunk 6]
```diff
@@ -1,29 +1,30 @@
...
 const getName=()=>{
     return "Nikhil"
 }
-const f=(name=getName())=>{
-    return "Hii" + " "+name
+const printName=(name=getName())=>{
+    return "Hii" + " "+name  
 }
-console.log(f())
+console.log(printName())
```
[the remark this answers is not in the document]
Comment (nikhil1-successive) [reply Assignment-2/Ques7.js]: Done

**Review by shahbaz-successive — APPROVED** [review]
👍

**Pull request merged**